test/clj/puppetlabs/trapperkeeper/services/webserver/jetty9_default_config_test.clj

```diff
@@ -143,15 +150,22 @@ react accordingly."
                                  (get-server-thread-pool-queue server)))
         "Unexpected default for 'queue-max-size'")))
 
-(def threads-per-connector
+(defn required-threads-for-sized-threadpool
+  [threadpool-size]
   "The total number of threads needed per attached connector."
-  (+ acceptor-thread-count selector-thread-count))
+  (+ (reserved-thread-count threadpool-size)
+     (selector-thread-count threadpool-size)
+     acceptor-thread-count))
```

Comment:
This doesn't seem to hold when the threadpool is super small and I don't know why.